examples/tof-viewer/src/ADIMainWindow.cpp

```diff
@@ -581,6 +584,53 @@ void ADIMainWindow::showRecordMenu() {
     }
 }
 
+void ADIMainWindow::showSaveAdsdParamsMenu() {
+
+    char pathname[512];
+    std::strcpy(pathname, saveConfigurationPath.c_str());
+
+    if (ImGui::BeginMenu("Save Configuration", view != nullptr)) {
+
+        ImGui::Text("File name:");
+        ImGui::InputText("", pathname, 300);
+        ImGui::NewLine();
+
+        { // Use block to control the moment when ImGuiExtensions::ButtonColorChanger gets destroyed
+            ImGuiExtensions::ButtonColorChanger colorChangerStartRec(
+                customColorPlay, isPlaying);
+            if (ImGuiExtensions::ADIButton("Save", true)) {
+
+                bool saveconfigurationFile = false;
+                std::string saveconfigurationFileValue = std::string(pathname);
+                if (!saveconfigurationFileValue.empty()) {
+                    if (saveconfigurationFileValue.find(".json") ==
+                        std::string::npos) {
+                        saveconfigurationFileValue += ".json";
+                    }
+                    saveconfigurationFile = true;
+                }
+                if (saveconfigurationFile && view) {
+                    auto camera =
+                        view->m_ctrl->m_cameras[static_cast<unsigned int>(
+                            view->m_ctrl->getCameraInUse())];
+
+                    aditof::Status status = camera->saveDepthParamsToJsonFile(
+                        saveconfigurationFileValue);
+                    if (status != aditof::Status::OK) {
+                        std::cout
+                            << "Could not save current configuration info to "
+                            << saveconfigurationFileValue << std::endl;
+                    } else {
+                        std::cout << "Current configuration info saved to file "
```

Comment:
Most of the logging if not all is done via GLOG() calls. Can you please use the same?